CONTRIBUTING.md

```diff
@@ -151,6 +152,11 @@ Validate authentication using the username `alice` with the password `alice`:
 $ curl -v -k -X POST -d "alice" http://localhost:3000/authn-ldap/test/cucumber/alice/authenticate
 ```
 
+#### Google Cloud Platform (GCP) Authentication
+
+To enable a host to log into Conjur using GCP identity token, run `start` with the `--authn-gcp` flag.
+Form more information on how to setup Conjur Google Cloud (GCP) authenticator, follow the official [documentation](https://www.conjur.org/). 
```

Comment:
I would add
"follow our official OSS documentation <link>"

do we want to link to open source only?
Do you have a link to the GCP authenticator? If so, please link to it?

Reply:
It's Conjur not DAP. We will have a link when shulli will create one.